docs/UserGuide.md

```diff
@@ -136,38 +157,64 @@ Format: `deleteCompany COMPANY_INDEX`
+* The index refers to the index number shown in the displayed company list.
+* The indexes must be a positive integer like 1, 2, 3, …
+
 Examples:
 
-* `list` followed by `deleteCompany 2` deletes the 2<sup>nd</sup> company in the displayed company list.
+* `list` followed by, `favourite 1` favourites the 1<sup>st</sup>
+  company in Tinner.
 
-### Deleting an internship role from company : `deleteRole` <a id="c-delete-c-r"></a>
+### Unfavouriting a specific company: `unfavourite` <a id="c-unfavourite-c"></a>
 
-Deletes the specified role in the specified company within the displayed company list.
+Unfavourite a specific company from the list of companies.
 
-Format: `deleteRole COMPANY_INDEX ROLE_INDEX`
+Format: `unfavourite COMPANY_INDEX`
 
-* Deletes the role at the specified `ROLE_INDEX` of the company at the specified `COMPANY_INDEX`. The indexes refer to the index numbers shown in the displayed company list.
+* Unfavourites the company at the specified `COMPANY_INDEX`.
+* The index refer to the index number shown in the displayed company list.
 * The indexes must be a positive integer like 1, 2, 3, …
 
 Examples:
 
-* `list` followed by, `deleteRole 1 1` deletes the 1<sup>st</sup> role from the 1<sup>st</sup>
-  company in the displayed company list.
+* `list` followed by, `unfavourite 1` unfavourites the 1<sup>st</sup>
+  company in Tinner.
 
+## Modifying Roles <a id="c-modifying-roles"></a>
 
-### Editing an existing company in the company list : `editCompany` <a id="c-edit-c"></a>
+### Adding an internship role to an existing company: `addRole` <a id="c-add-c-r"></a>
 
-Edits the specified company within the displayed company list.
+Adds an internship role to a company that already exists in the list.
 
-Format: `editCompany COMPANY_INDEX [n/COMPANY_NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS]`
+Format: `addRole COMPANY_INDEX n/ROLE_NAME [(TYPE)] s/STATUS r/REMINDER_DATE [d/DESCRIPTION] [$/STIPEND]`
+* Add internship role at the specified `COMPANY_INDEX`.
+* The `COMPANY_INDEX` must be a positive integer like 1, 2, 3, ...
+* The `ROLE_NAME` should only contain alphanumeric characters, spaces and an optional pair of round brackets.
+* The `STATUS` accepted are as follows: applying, pending, interview and assessments, offered, rejected, complete.
+* The `REMINDER_DATE` should be in format dd-MM-yyyy HH:mm.
+* The `DESCRIPTION` and `STIPEND` fields are optional during the initial role creation.
+    * The `DESCRIPTION` can contain alphanumeric characters, spaces and special characters.
+    * The `STIPEND` must be a positive integer going up to 10 digits long.
 
-* Edits the company at the specified INDEX. The index refers to the index number shown in the displayed company list. The index must be a positive integer 1, 2, 3, …​
-* At least one of the optional fields must be provided.
-* Existing values will be updated to the input values.
+Note: Integer is a number which is not a fraction; a whole number.
```

Comment:
I thinking for generic terms like integers and parameters that will appear throughout the UG can put at technical terminologies.

Reply:
Sure. Will add and consolidate in later iterations.